.circleci/soltest.sh

```diff
@@ -38,6 +38,9 @@ OPTIMIZE=${OPTIMIZE:-"0"}
 EVM=${EVM:-"invalid"}
 REPODIR="$(realpath "$(dirname "$0")/..")"
 
+IFS=" " read -r -a BOOST_TEST_ARGS <<< "$BOOST_TEST_ARGS"
+IFS=" " read -r -a SOLTEST_FLAGS <<< "$SOLTEST_FLAGS"
```

Comment:
is IFS needed?

Reply:
I included it because that's the form recommended by shellcheck ([SC2207](https://github.com/koalaman/shellcheck/wiki/SC2207)). The default for `IFS` is to split on spaces, tabs and newlines. This changes the behavior to splitting only on spaces. Tabs and newlines would be weird in these variables so I think either is fine and I went with it. With the general shell weirdness I wouldn't be surprised if it there was some obscure side-effect to leaving it at the default. But I can try to remove it and see what happens if you think we're better off without it.

Reply:
Ok, I'm fine either way, just wanted to ask